frontend/src/static/js/components/test/webstatus-notfound-error-page.test.ts

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {expect, fixture, html} from '@open-wc/testing';
+import '../webstatus-not-found-error-page.js';
```

Comment:
```suggestion
import '../webstatus-notfound-error-page.js';
```

This is the typo that you have causing the test file to not run. After you fix it, you will need to look at the failing tests.

Reply:
Hey @jcscottiii — I was away due to some eye strain.

The test failures were due to a misunderstanding on my part — I had assumed unit tests would call the real API, but they required mocking. I’ve now updated the component with clearer naming and refactored the tests to use a reusable mock API function.

Precommit checks are all passing now. I believe it’s ready for review!

Let me know if anything else is needed 🙌